emulator/src/extensions/emu_args.c

```diff
@@ -172,6 +173,7 @@ static const optDoc_t argDocs[] =
     { 0,  argFuzzMotion,  "y|n",   "Set whether motion inputs are fuzzed" },
     { 0,  argHeadless,    NULL,    "Runs the emulator without a window." },
     { 0,  argHideLeds,    NULL,    "Don't draw simulated LEDs next to the display" },
+    {'k', argKeymap,     "LAYOUT", "Use an alternative keymap. LAYOUT can be azerty, colemak, dvorak, or dvp"},
```

Comment:
Is querty the same as dvp?

```c
static const emuKeymap_t keymaps[] = {
    {.name = "qwerty", .keymap = "WSADLKOI"},  // QWERTY (default)
    {.name = "azerty", .keymap = "ZSQDLKOI"},  // AZERTY
    {.name = "colemak", .keymap = "WRASIEYU"}, // Colemak
    {.name = "dvorak", .keymap = ",OAENTRC"},  // Dvorak
};
```

Reply:
Ah, I removed dvp earlier because I realized it was now the same as regular dvorak without the touchpad keys. I'll remove it from the help too.